### ISSUE TITLE
Visual Studio 2017 support?

### ISSUE THREAD

**Issue by jschneid** [IPWright83/VSFileNav#17]
Any chance of updating this extension to support Visual Studio 2017?  I'm on day one of using VS 2017 and I already miss VSFileNav like crazy!

I tried using the existing installer from https://marketplace.visualstudio.com/items?itemName=IPWright83.VSFileNav-17743, but the VSIX installer reported "This extension is already installed to all applicable products."

Thanks!  (Let me know if I can do anything to help!)

**Comment by acki-m** [comment]
+1

**Comment by zack-snyder** [comment]
+1

**Comment by IPWright83** [comment]
@jschneid thanks for the request and sorry for the delay in replying. I believe the manifest file has changed (again) for VS2017 which makes it a bit of a pain. (Last time they did that I never managed to reconcile the VS2010 and VS2012 installers). I had set it up in theory to support all future versions, but it never seems to work.

I'm currently installing VS2017 so might be able to give this a try, but I'd happy for you to change yourself and issue a pull request. It all comes down to the 
`vsixmanifest` file - this controls the versions of Visual Studio for which it will install. I'd imagine VS2017 provides a helpful form to just check new versions.

**Comment by jschneid** [comment]
@IPWright83 Ok! I took a stab at it, and I have a new branch with Visual Studio 2017 working in my local environment!

This would be my first pull request on someone else's project, though, and I need a bit of help! I tried uploading (pushing) my branch to use in a new pull request, but got a 403 Forbidden error back from GitHub.  Is there something I'm doing wrong?  Or is some configuration change needed to the project?

Update: I just read http://stackoverflow.com/questions/29044551/http-403-on-github-pull-request and it looks like the issue is I was supposed to fork the repository, and then make my changes there. I'll give that a shot.
